### PR TITLE
Release 31.2

### PR DESCRIPTION
**Pull request by stevemessick**
Update change log with latest work for M31.2.

@devoncarew @pq @jacob314 @terrylucas @kenzieschmoll

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## 31.2
+- show memory profiler legend with proper line chart color or line style
+- prevent the (IntelliJ) New Project wizard from completing when there is no Flutter SDK
+- fix a race condition causing unexpected conditions in attach
+- added control of RSS display to memory profiler
+- when running the flutter doctor command, use the -v flag
+- add https_proxy to the readme
```
Comment (pq) [inline CHANGELOG.md]: I'd maybe drop this since it's not plugin-facing.

Comment (stevemessick) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## 31.2
+- show memory profiler legend with proper line chart color or line style
```
Comment (devoncarew) [inline CHANGELOG.md]: There was also a fix for the hot restart action with used in IntelliJ 2018.3.

**Comment by stevemessick** [comment]
Merging as-is; can update later if needed.

File: CHANGELOG.md (hunk 2 above)
Comment (devoncarew) [reply CHANGELOG.md]: This was @pq's PR: https://github.com/flutter/flutter-intellij/pull/2895

Comment (stevemessick) [reply CHANGELOG.md]: That PR was included in M31.1.

**Pull request merged**